src/com/company/ppj09/Tasks.java

```diff
@@ -0,0 +1,58 @@
+package com.company.ppj09;
+
+import java.util.Scanner;
+
+public class Tasks {
+    public static void main(String[] args) {
+        //task01();
+        //task02();
+        //task03();
+        //task04();
+    }
+
+    private static void task01() {
+        for (int i = 75; i < 150; i++) {
+            System.out.println((char) i + " - " + i);
+        }
+    }
+
+    private static void task02() {
+        for (int i = 1; i <= 10; i++) {
+            System.out.println(1 / Math.pow(2, i));
+        }
+    }
+
+    private static void task03() {
+        for (int i = 1; i <= 15; i++) {
+            convertByteToHexAndSaveAsChar((byte) i);
+        }
+    }
+
+    private static void convertByteToHexAndSaveAsChar(byte a) {
+        String hexadecimalValue = String.format("%02x", a);
```

Comment:
this code doesn't work, use `char ch = Integer.toHexString(a).charAt(0);` instead

Reply:
fixed